vms/platformvm/txs/fee/static_config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fee
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/vms/platformvm/upgrade"
+)
+
+type StaticConfig struct {
+	// Fee that is burned by every non-state creating transaction
+	TxFee uint64
+
+	// Fee that must be burned by every state creating transaction before AP3
+	CreateAssetTxFee uint64
+
+	// Fee that must be burned by every subnet creating transaction after AP3
+	CreateSubnetTxFee uint64
+
+	// Fee that must be burned by every transform subnet transaction
+	TransformSubnetTxFee uint64
+
+	// Fee that must be burned by every blockchain creating transaction after AP3
+	CreateBlockchainTxFee uint64
+
+	// Transaction fee for adding a primary network validator
+	AddPrimaryNetworkValidatorFee uint64
+
+	// Transaction fee for adding a primary network delegator
+	AddPrimaryNetworkDelegatorFee uint64
+
+	// Transaction fee for adding a subnet validator
+	AddSubnetValidatorFee uint64
+
+	// Transaction fee for adding a subnet delegator
+	AddSubnetDelegatorFee uint64
+
+	// The minimum amount of tokens one must bond to be a validator
+	MinValidatorStake uint64
+
+	// The maximum amount of tokens that can be bonded on a validator
+	MaxValidatorStake uint64
+
+	// Minimum stake, in nAVAX, that can be delegated on the primary network
+	MinDelegatorStake uint64
+
+	// Minimum fee that can be charged for delegation
+	MinDelegationFee uint32
+}
+
+func (c *StaticConfig) GetCreateBlockchainTxFee(upgrades upgrade.Times, timestamp time.Time) uint64 {
+	if upgrades.IsApricotPhase3Activated(timestamp) {
+		return c.CreateBlockchainTxFee
+	}
+	return c.CreateAssetTxFee
+}
+
+func (c *StaticConfig) GetCreateSubnetTxFee(upgrades upgrade.Times, timestamp time.Time) uint64 {
+	if upgrades.IsApricotPhase3Activated(timestamp) {
+		return c.CreateSubnetTxFee
+	}
+	return c.CreateAssetTxFee
+}
```

Comment:
I feel like these are weird to have in `StaticConfig` now that we're introducing the calculator as the place to determine fees. Can we just get rid of these functions and in-line them there?

Reply:
The advantage is that I can get the right fee without instantiating a calculator.
Also they don't err, while the GetFee method will err in next PRs, so I get some code simplification by keeping them here, since I won't need to bubble up an err that I know can't happen with these fees.
I'd vote to keep them, but happy to rework it if consensus is to do everything through the calculator.